k2/csrc/fsa_renderer.h

```diff
@@ -16,13 +16,14 @@ namespace k2 {
 // Get a GraphViz representation of an fsa.
 class FsaRenderer {
  public:
-  explicit FsaRenderer(const Fsa &fsa);
+  explicit FsaRenderer(const Fsa &fsa, float *arc_weights = nullptr);
```

Comment:
`const float *`

`FsaRenderer` is not supposed to change `arc_wegiths`, so it's better to add `const`.

Reply:
Sure